runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -725,6 +726,34 @@ impl<'a> VMLogic<'a> {
         )
     }
 
+    /// Saves the account id that was set for the current receipt by its
+    /// predecessor via [`Self::promise_set_refund_to()`], or
+    /// [`Self::predecessor_account_id()`] otherwise, into the register.
```

Comment:
```suggestion
    /// Populates a register with the ID of an account which would receive a refund.
    ///
    /// This is the ID of an account set for the current receipt by its
    /// predecessor via [`Self::promise_set_refund_to()`], or
    /// [`Self::predecessor_account_id()`] otherwise.
```